#A representation of a multi-dof joint state
time stamp
string joint_name
string frame_id
string child_frame_id
geometry_msgs/Pose pose
